templates/policies/privacy.hbs

```diff
@@ -0,0 +1,204 @@
+{{#*inline "page"}}
+
+<header class="mt3 mt2-ns mb4 mb5-ns tc tl-ns">
+  <div class="w-100 mw-none ph3 mw8-m mw9-l center f3">
+    <h1>{{text policies-privacy-page-title}}</h1>
+  </div>
+</header>
+
+<section id="privacy-notice" class="purple">
+  <div class="w-100 mw-none ph3 mw8-m mw9-l center f3">
+    <p>Version 1.0, dated <time datetime="2019-08-19">2019-08-19</time></p>
+
+    <p>
+      The Rust Project oversees a number of projects, including the
+      <a href="{{baseurl}}/">Rust website</a>,
+      <a href="https://crates.io">crates.io</a>, and
+      <a href="https://docs.rs">docs.rs</a>. This privacy notice explains what
+      we do with personal information.
+    </p>
+
+    <header>
+      <h2 id="rust-www">rust-lang.org</h2>
+      <div class="highlight"></div>
+    </header>
+
+    <p>
+      The Rust website at <a href="{{baseurl}}/">rust-lang.org</a> is a project
+      of the <a href="{{baseurl}}/governance/teams/core">Core team</a> and the
+      <a href="{{baseurl}}/governance/teams/community">Community team.</a>
+    </p>
+
+    <dl>
+      <dt>Visitor logs:</dt>
+      <dd>
+        When you visit rust-lang.org, we receive your IP address as part of
+        our standard server logs. We store these logs for 1 year.
```

Comment:
Does this also include static.rust-lang.org? Because we store for 1 years the logs of that as well. Not sure we should keep doing that, as I don't see much benefit from having those logs.

Reply:
@pietroalbini i think i saw you mentioning in discord that you adjusted the retention policy for these? is that right?

Reply:
That's for static.rust-lang.org and static.crates.io, not the website itself. I don't know if the retention policy for the website changed.